source/reference/modules/engine/index.rst

```diff
@@ -1330,6 +1330,18 @@ Crafter Engine sports a built-in cache engine with an LRU (least recently used)
 
 .. note:: When running in Preview Mode (inside Studio for preview purposes), Crafter Engine's cache is disabled to help authors see their changes immediately.
 
+Engine has multiple caches:
+
+.. version_tag::
+    :label: Since
+    :version: 4.3.1
+
+- **user** cache for objects cached by Groovy code.
```

Comment:
`application` cache. It's enabled by specifying the value `crafter.core.applicationCache` in the `SPRING_PROFILES_ACTIVE` env variable